crates/oxc_linter/src/rules/jest/prefer_to_contain.rs

```diff
@@ -0,0 +1,389 @@
+            KnownMemberExpressionParentKind::Call
+        ) || jest_expect_fn_call.args.is_empty()
+        {
+            return;
+        }
+
+        let Some(Argument::Expression(jest_expect_first_arg)) = jest_expect_fn_call.args.first()
+        else {
+            return;
+        };
+        let Expression::BooleanLiteral(bool_literal) = jest_expect_first_arg.get_inner_expression()
+        else {
+            return;
+        };
+        let Expression::CallExpression(expect_call_expr) = parent else {
+            return;
+        };
+
+        // handle "expect()"
+        if expect_call_expr.arguments.is_empty() {
+            return;
+        }
+
+        let first_argument = expect_call_expr.arguments.first().unwrap();
+        let Argument::Expression(Expression::CallExpression(includes_call_expr)) = first_argument
+        else {
+            return;
+        };
+
+        if !is_equality_matcher(matcher)
+            || !Self::is_fixable_includes_call_expression(includes_call_expr)
+        {
+            return;
+        }
+
+        Self::report_and_fix(
+            bool_literal.0,
+            includes_call_expr,
+            &jest_expect_fn_call,
+            matcher.span,
+            call_expr.span,
+            ctx,
+        );
+    }
+
+    fn is_fixable_includes_call_expression(call_expr: &CallExpression) -> bool {
+        let Expression::MemberExpression(mem_expr) = &call_expr.callee else {
+            return false;
+        };
+
+        mem_expr.static_property_name() == Some("includes")
+            // handle "expect(a.includes())"
+            && !call_expr.arguments.is_empty()
+            // handle "expect(a.includes(b,c))"
+            && call_expr.arguments.len() == 1
+            // handle "expect(a.includes(...[]))"
+            && matches!(call_expr.arguments.first(), Some(Argument::Expression(_)))
+    }
+
+    fn report_and_fix(
+        bool_arg: &BooleanLiteral,
+        call_expr: &CallExpression,
+        jest_expect_fn_call: &ParsedExpectFnCall,
+        matcher_span: Span,
+        call_span: Span,
+        ctx: &LintContext,
+    ) {
+        let (includes_call_span, _) =
+            call_expr.callee.get_member_expr().unwrap().static_property_info().unwrap();
+        let arg = call_expr.arguments.first().unwrap();
+        let Argument::Expression(argument_expr) = arg else {
+            return;
+        };
+        let arg_span = match argument_expr {
+            Expression::ArrayExpression(arr_expr) => arr_expr.span,
+            Expression::BooleanLiteral(literal) => literal.span,
+            Expression::Identifier(ident) => ident.span,
+            Expression::StringLiteral(literal) => literal.span,
+            Expression::NumericLiteral(literal) => literal.span,
+            Expression::ObjectExpression(obj_expr) => obj_expr.span,
+            Expression::CallExpression(call_expr) => call_expr.span,
+            _ => unreachable!("Should upgrade to support this unknown type"),
```

Comment:
I think we can just report here without fix.

Reply:
Ok. But I have an extra question, how to judge is there need a fixer or not? Because some suggestion rules also has a fixer and some not.

Reply:
I apologize for speaking something ambiguous, I think you write a good fixer for this rule! My thought is if we can replace the `unreachable` statement with a diagnostic (no fixer is okay), people may write the right code for them, but this code lets our linter panic, this may unexpected for them.

Reply:
Thank you for your answer. I removed the fixer. Currently it's difficult to check all the Expressions indeed.😬